hack/apt/build_package_repo.sh

```diff
@@ -42,6 +42,21 @@ if ! command -v bzip2 &> /dev/null || ! command -v curl &> /dev/null; then
    ${PKG_MGR} install -y bzip2 curl
 fi
 
+function cleanPackageFile {
+   file=$1
+
+   # There must be an empty line between each package section.
+   # Our package sections alwaya end with the "Description: " line,
+   # so let's append an empty line after that Description line.
```

Comment:
> let's append an empty line after that Description line.

 strictly this is more about appending an empty line after EVERY "Description: " line.